students/nartowska_karina/lesson_05_lists/age_calculator.py

```diff
@@ -0,0 +1,41 @@
+def average_age(values):
+    average = 0
+    adult = 0
+    for i in values:
+        if i >= 18:
+            average += i
+            adult += 1
+    if adult > 0:
+        print("The average age of adults: ")
+        return average / adult
+    else:
+        print("The average age of adults: ")
```

Comment:
Both prints from line 9 and 12 could be placed once in main() function, so this function will only calculate the result.